DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -190,9 +203,65 @@ void L1TStage2uGT::analyze(const edm::Event& evt, const edm::EventSetup& evtSetu
             }
          }
       }
-   }
-}
 
+	for(auto itr = uGtAlgs->begin(0); itr != uGtAlgs->end(0); ++itr) { 
```

Comment:
Hi @dtsitson ,
please add some proper indentation to this code and use spaces instead of tabulators.